Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10.0-buster
+FROM python:buster
```

Comment:
this should be put to:
```
FROM python-3.10:buster
```

to avoid unattended bumps to 3.11 or future stable releases